bf_cpp_downloads.html

```diff
@@ -300,7 +379,14 @@
                                         href="http://github.com/openmicroscopy/bioformats"
                                         target="_blank"><img
                                         src="images/bf_gh_repo.png"
-                                        alt="GitHub Repository Link" /></a></td>
+                                        alt="Bio-Formats GitHub Repository Link" /></a></td>
+                            </tr>
+                            <tr>
+                                <td><a
+                                        href="http://github.com/ome/ome-cmake-superbuild"
+                                        target="_blank"><img
+                                        src="images/bf_gh_repo.png"
+                                        alt="CMake Super-Build GitHub Repository Link" /></a></td>
                             </tr>
                             <tr>
                                 <td><a href="http://git.openmicroscopy.org/"
```

Comment:
images/bf_gh_web.png (line 394 below) needs changing to images/gitweb_interface.png